lib/src/v2/crowd_notifier_primitives.ts

```diff
@@ -26,6 +28,47 @@ export function setupHA(): IKeyPair {
   return crypto_box_keypair();
 }
 
+/**
+ * Generates the static part for v2.1 from a pass phrase. The idea is to
+ * have a static part that doesn't change, to secure the notifications without
+ * having to keep too many keys.
+ * @param pkh public key of the health authority
+ * @param pp pass phrase, if possible 256 bits entropy or more
+ */
+export function genOrgStatic(pkh: Uint8Array, pp: string): IOrganizerData {
+  const mskO = new mcl.Fr();
+  mskO.setHashOf(from_string(pp));
```

Comment:
This should probably be something a bit more secure, such as BPKDF2 or something of the sort.

Reply:
There is no such npm :(

I'm currently creating a random 256-bit hex string as a pass phrase in the web UI, so I don't think BPKDF2 will change anything there. Are you OK if I just increase disclaimer in the `genorgStatic` method? Currently it says:

```
 * @param pp pass phrase, if possible 256 bits entropy or more
```

So I could beef it up to something like:

```
 * @param pp pass phrase, needs to have at least 256 bit of entropy. E.g.: 
 * Buffer.from(randomBytes(32)).toString('hex')
```